src/go/intelligence/int_test/test_wrapper.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/sh
+##H Script for automation of testing process of the intelligence module.
+##H Usage: test_wrapper.sh <work_directory>
+##H
+
+# If user provided the input parameter
+if [ $# -ne 1 ]; then
+    perl -ne '/^##H/ && do { s/^##H ?//; print }' <$0
+    exit 1
+fi
+
+# Setup work directory based on user input
+WDIR=$1
```

Comment:
Better replace with
```
WDIR=${1:-"/tmp/$USER"}
```
which will allow to have default in /tmp area for each individual user. Please test though and if it works make adjustment to doc script at the beginning of the script to specify the default work dir